pom.xml

```diff
@@ -137,6 +145,11 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>${maven.surefire.plugin.version}</version>
         <configuration>
+          <!-- If we don't set forkMode to 'always', a Mockito error will appear in Maven saying that it can't mock certain 
+            classes.
+            This error only appears in Maven (not Eclipse)
+            TODO: find a way to run all the tests successfully in Maven -->
```

Comment:
Worth making a ticket for this so its not forgotten?

Reply:
Maybe, although the ticket might look like 'wait for Mockito to fix the bug'. @massdosage what do you think?

Reply:
I think we should reach out to the Mockito developer list/gitter/whatever and describe the problem and ask for their thoughts, based on that we can decide what we're going to do. @shermosa please create an issue in this repo and outline the steps to reproduce it, the stack trace of the exception etc. and we can then use that as the basis for reaching out to the Mockito devs.